scipy/optimize/_minimize.py

```diff
@@ -56,9 +56,12 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     Parameters
     ----------
     fun : callable
-        Objective function.
+        Objective function. Takes a vector of variables as its only argument,
```

Comment:
My version is more precise. Also, `fun` accepts additional arguments, specified by `args`.